api/src/main/kotlin/pl/edu/pk/backend/controller/TicketController.kt

```diff
@@ -0,0 +1,49 @@
+package pl.edu.pk.backend.controller
+
+import io.vertx.ext.web.RoutingContext
+import pl.edu.pk.backend.model.Role
+import pl.edu.pk.backend.service.TicketService
+import pl.edu.pk.backend.util.ApiError
+
+class TicketController(private val ticketService: TicketService) {
+  fun getTickets(ctx: RoutingContext) {
+    if (!ctx.checkCurrentUserHasRole(Role.User)) return
+    var all = "false"
+    try {
+      all = ctx.queryParam("all")[0]
+    } catch (ex: Exception) {
+    }
```

Comment:
Don't leave empty catch blocks and avoid using exceptions for normal control flow. 
You can rewrite this using:
```
ctx.queryParam("all").firstOrNull()?.toBoolean()
```